src/main/java/model/user/Participants.java

```diff
@@ -40,8 +33,41 @@ private List<Score> createFinalResultWithoutDealer(final int dealerTotalValue) {
     public void receiveInitialCards(final Deck deck) {
         players.forEach(player -> player.receiveInitialCards(deck));
         dealer.receiveInitialCards(deck);
+        ifPlayerBlackJackReceiveMoney();
     }
 
+    public void ifPlayerBlackJackReceiveMoney() {
+        boolean dealerResult = dealer.isBlackJack();
+        for (Player player : players) {
+            if (player.isBlackJack() && !dealerResult) {
+                player.receiveMoney();
+            }
+        }
+    }
+
+    public int getDealerProfit() {
+        List<Score> finalResult = getFinalResult();
+        int dealerMoney = 0;
+        for (int i = 0, size = finalResult.size(); i < size; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < finalResult.size(); i++) {
```

요렇게 바꾸면 베베가 생각하기에 가독성이 더 좋은가요? ㅎㅎ

Reply:
아! 이 부분은 for문이 실행될 때 마다 finalResult의 size를 재호출하는 문제가 있다고 해서 미리 size를 선언해놓았습니다!